data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```diff
@@ -20,7 +20,7 @@
             {PROLOGUE_STAGE1}
         [/part]
         [part]
-            story= _ "The dwarves of Knalga, cannily reckoning the gains from trade, built strong fortifications around Dwarven Doors. The people who called it home, dwarven and human alike, knew that the wealth of their city might arouse envy from afar — hence they kept their weapons sharp, and reckoned themselves well able to fight off any bandit gang or petty warlord that could arise in the thinly-settled Northlands."
+            story= _ "The dwarves of Knalga, cannily reckoning the gains from trade, built strong fortifications around Dwarven Doors. The people who called it home, dwarven and human alike, knew that the wealth of their city might arouse envy from afar — hence they kept their weapons shar and were well able to fight off any bandit, gang, or petty warlord that could arise in the thinly-settled Northlands."
```

Comment:
now the "p" is missing from "sharp", though

Reply:
Oops.

Reply:
Oops I fixed it. I'm not perfect either.